bundles/org.openhab.binding.guntamatic/README.md

```diff
@@ -0,0 +1,362 @@
+|	`programHc4`	|	Program HC4	|	`String`	|		|	🔓 W0	|	R/O	|	false	|
+|	`programHc5`	|	Program HC5	|	`String`	|		|	🔓 W0	|	R/O	|	false	|
+|	`programHc6`	|	Program HC6	|	`String`	|		|	🔓 W0	|	R/O	|	false	|
+|	`programHc7`	|	Program HC7	|	`String`	|		|	🔓 W0	|	R/O	|	false	|
+|	`programHc8`	|	Program HC8	|	`String`	|		|	🔓 W0	|	R/O	|	false	|
+|	`interuption0`	|	Interuption 0	|	`String`	|		|	🔓 W0	|	R/O	|	false	|
+|	`interuption1`	|	Interuption 1	|	`String`	|		|	🔓 W0	|	R/O	|	false	|
+|	`serial`	|	Serial	|	`Number`	|		|	🔓 W0	|	R/O	|	false	|
+|	`version`	|	Version	|	`String`	|		|	🔓 W0	|	R/O	|	false	|
+|	`runningTime`	|	Running Time	|	`Number:Time`	|	`h`	|	🔓 W0	|	R/O	|	false	|
+|	`serviceHrs`	|	Service Hrs	|	`Number:Time`	|	`d`	|	🔓 W0	|	R/O	|	false	|
+|	`emptyAshIn`	|	Empty ash in	|	`Number:Time`	|	`h`	|	🔓 W0	|	R/O	|	false	|
+|	`flowIs0`	|	Flow is 0	|	`Number:Temperature`	|	`°C`	|	🔓 W0	|	R/O	|	false	|
+|	`flowIs3`	|	Flow is 3	|	`Number:Temperature`	|	`°C`	|	🔓 W0	|	R/O	|	false	|
+|	`flowIs6`	|	Flow is 6	|	`Number:Temperature`	|	`°C`	|	🔓 W0	|	R/O	|	false	|
+|	`fuelCounter`	|	Fuel counter	|	`Number:Volume`	|	`m³`	|	🔐 W1	|	R/O	|	false	|
+|	`bufferLoad`	|	Buffer load.	|	`Number:Dimensionless`	|	`%`	|	🔓 W0	|	R/O	|	false	|
+|	`bufferTop0`	|	Buffer Top 0	|	`Number:Temperature`	|	`°C`	|	🔐 W1	|	R/O	|	false	|
+|	`bufferBtm0`	|	Buffer Btm 0	|	`Number:Temperature`	|	`°C`	|	🔐 W1	|	R/O	|	false	|
+|	`bufferTop1`	|	Buffer Top 1	|	`Number:Temperature`	|	`°C`	|	🔐 W1	|	R/O	|	false	|
+|	`bufferBtm1`	|	Buffer Btm 1	|	`Number:Temperature`	|	`°C`	|	🔐 W1	|	R/O	|	false	|
+|	`bufferTop2`	|	Buffer Top 2	|	`Number:Temperature`	|	`°C`	|	🔐 W1	|	R/O	|	false	|
+|	`bufferBtm2`	|	Buffer Btm 2	|	`Number:Temperature`	|	`°C`	|	🔐 W1	|	R/O	|	false	|
+|	`bExtraWw0`	|	B extra-WW. 0	|	`Switch`	|		|	🔐 W1	|	R/O	|	false	|
+|	`bExtraWw1`	|	B extra-WW. 1	|	`Switch`	|		|	🔐 W1	|	R/O	|	false	|
+|	`bExtraWw2`	|	B extra-WW. 2	|	`Switch`	|		|	🔐 W1	|	R/O	|	false	|
+|	`auxiliaryPump0`	|	Auxiliary pump 0	|	`Switch`	|		|	🔐 W1	|	R/O	|	false	|
+|	`auxiliaryPump1`	|	Auxiliary pump 1	|	`Switch`	|		|	🔐 W1	|	R/O	|	false	|
+|	`auxiliaryPump2`	|	Auxiliary pump 2	|	`Switch`	|		|	🔐 W1	|	R/O	|	false	|
+|	`boilersConditionNo`	|	Boiler´s condition no.	|	`String`	|		|	🔐 W1	|	R/O	|	false	|
+|	`bufferT5`	|	Buffer T5	|	`Number:Temperature`	|	`°C`	|	🔓 W0	|	R/O	|	false	|
+|	`bufferT6`	|	Buffer T6	|	`Number:Temperature`	|	`°C`	|	🔓 W0	|	R/O	|	false	|
+|	`bufferT7`	|	Buffer T7	|	`Number:Temperature`	|	`°C`	|	🔓 W0	|	R/O	|	false	|
+|	`extraWw0`	|	Extra-WW. 0	|	`Number:Temperature`	|	`°C`	|	🔓 W0	|	R/O	|	false	|
+|	`extraWw1`	|	Extra-WW. 1	|	`Number:Temperature`	|	`°C`	|	🔓 W0	|	R/O	|	false	|
+|	`extraWw2`	|	Extra-WW. 2	|	`Number:Temperature`	|	`°C`	|	🔓 W0	|	R/O	|	false	|
+|	`grate`	|	Grate	|	`Number:Dimensionless`	|	`%`	|	🔓 W0	|	R/O	|	false	|
+
+#### Security Access Levels
+
+- 🔓 W0 ... Open
+- 🔐 W1 ... End Customer Key
+- 🔒 W2 ... Service Partner
+
+## Full Example
+
+**Thing File**
+
+```java
```

Comment:
I believe you do not need to mention "java"

Reply:
> I believe you do not need to mention "java"

I've seen this in other README's.
With `java`, a proper syntax highlighting is used.
Without `java`, the text is rendered in black only.